charts/seed-bootstrap/charts/loki/templates/statefulset.yaml

```diff
@@ -16,15 +16,66 @@ spec:
     metadata:
       labels:
         networking.gardener.cloud/from-prometheus: allowed
+{{- if .Values.rbacSidecarEnabled }}
+        networking.gardener.cloud/to-dns: allowed
+        networking.gardener.cloud/to-shoot-apiserver: allowed
+{{- end}}
 {{ toYaml .Values.labels | indent 8 }}
       annotations:
         checksum/configmap-loki-config: {{ include (print $.Template.BasePath "/loki-configmap.yaml") . | sha256sum }}
+{{- if .Values.rbacSidecarEnabled }}
+        checksum/kube-rbac-proxy-kubeconfig: {{ .Values.kubeRBACProxyKubeconfigCheckSum }}
+        checksum/configmap-telegraf-config: {{ include (print $.Template.BasePath "/telegraf-config.yaml") . | sha256sum }}
+{{- end}}
     spec:
       serviceAccountName: loki
       securityContext:
-        {{- toYaml .Values.securityContext | nindent 8 }}
+        fsGroup: 10001
       priorityClassName: {{ .Values.priorityClass.name }}
       containers:
+{{- if .Values.rbacSidecarEnabled }}
+        - name: kube-rbac-proxy
+          image: {{ index .Values.global.images "kube-rbac-proxy" }}
+          args:
+          - --insecure-listen-address=0.0.0.0:8080
+          - --upstream=http://127.0.0.1:3100/
+          - --kubeconfig=/kubeconfig/kubeconfig
+          - --logtostderr=true
+          - --v=6
+          ports:
+          - name: kubeproxy
+            containerPort: {{ .Values.kubeProxy.port }}
+            protocol: TCP
+          volumeMounts:
+            - name: kubeconfig
+              mountPath: /kubeconfig
+          securityContext:
+{{- toYaml .Values.securityContext | nindent 12 }}
+        - image: {{ index .Values.global.images "telegraf" }}
```

Comment:
Same for the resources

Reply:
Same as above. Are you agree with the resources.